mw-webapp/src/component/table/columns.tsx

```diff
@@ -1,10 +1,14 @@
-import {createColumnHelper, ColumnDef, CellContext} from "@tanstack/react-table";
+import {CellContext, ColumnDef, createColumnHelper} from "@tanstack/react-table";
+import {CurrentProblem} from "src/model/businessModel/CurrentProblem";
 import {DayReport} from "src/model/businessModel/DayReport";
 import {JobDone} from "src/model/businessModel/JobDone";
 import {PlanForNextPeriod} from "src/model/businessModel/PlanForNextPeriod";
-import {CurrentProblem} from "src/model/businessModel/CurrentProblem";
+import {DateUtils} from "src/utils/DateUtils";
 import styles from "src/component/table/columns.module.scss";
 
+const FIRST_INDEX = 0;
+const START_VALUE_ACCUM = 0;
```

Comment:
it is startValueAccum if you look at it like on parameter, but you should look on it here like on argument

Reply:
Rename to INITIAL_VALUE. Is it ok?

Reply:
no

Reply:
Rename to DEFAULT_VALUE

Reply:
Every default variable cod be named as default value. Meaningless

Reply:
DEFAULT_SUMMARY_TIME